src/socket/src/socket.rs

```diff
@@ -7,8 +7,7 @@ use fluvio_protocol::api::RequestMessage;
 use fluvio_protocol::api::ResponseMessage;
 
 use fluvio_future::net::{
-    BoxReadConnection, BoxWriteConnection, ConnectionFd, DefaultTcpDomainConnector,
-    TcpDomainConnector, TcpStream,
+    BoxReadConnection, BoxWriteConnection, ConnectionFd, DefaultDomainConnector, TcpDomainConnector,
```

Comment:
TcpDomainConnector should be only imported for Unix?

Reply:
I think I forgot to rename this to `SocketDomainConnector` or something in https://github.com/infinyon/future-aio/pull/70. `DefaultConnector is a `Box<dyn TcpDomainConnector>`.